AutomatticTracks/build.gradle

```diff
@@ -63,10 +63,13 @@ afterEvaluate {
     publishing {
         publications {
             maven(MavenPublication) {
-                artifactId 'Automattic-Tracks-Android'
-                description 'Android client for Nosara tracks (event tracking and analytics)'
                 from components.release
+
                 groupId 'com.automattic'
+                artifactId 'Automattic-Tracks-Android'
+                // version is set by 'publish-to-s3' plugin
+
+                description 'Android client for Nosara tracks (event tracking and analytics)'
                 pom {
                     licenses {
```

Comment:
We don't typically have `description` or `licenses` sections in our POMs. I am not sure what would be the best course of action in this case, so I kept the details of this publication but didn't add the same sections to the `experimentation` module to follow our other projects.

We could consider adding these sections to all projects, but I don't see the need for that yet. I could be convinced otherwise 😅

Reply:
> We could consider adding these sections to all projects, but I don't see the need for that yet. 

+1

This is open source code that anyone can access. Having a description would be helpful and good open source behavior.

At the same time, it's highly unlikely anyone outside Automattic would use our open source tools, because they're tailored to our usage. E.g. To use this library one would need to have their own Tracks instance running and I don't even know if there's a way to configure the base URL to which the library sends event.

Because of that, I'd be leaning towards focusing on more impactful work than adding descriptions to our projects.

Reply:
Just to add a little more context about this:

I might be mistaken in this, but I assume the description in the POM file is used by maven repositories (such as `mavenCentral`) so it's available for the project page in their site. We don't have such a situation, because we use our own Maven repository and we don't have project pages. So, I don't think it's actually useful to have the description or the license in the POM file.

Having said that, if and when we realize it's useful for something particular, it'll be easier to tailor these details to match that need.